IdentityServer/appsettings.json

```diff
@@ -22,6 +22,13 @@
             "FhictRedirectUri": "",
             "FhictScopes": ""
         },
+        "FfhictOIDC": {
+            "Authority": "https://identity.fhict.nl", 
+            "ClientId": "digitalexcellence", 
+            "ClientSecret": "BHvuhf4k94Q8vJ64UqVu7g9Cs4uD6yKr",
```

Comment:
This document contains the secret. As it is secret, I don't think that this should be in this (public) repository.
The secret should be loaded by the production environment another way (For example environment variables). This secret must be removed from this repo and revoked by the identity provider.

Reply:
this was already noted and they will be removed on squash, this is needed to test the feature.

Reply:
So this is not the actual secret used in the production environment?

Reply:
Anyway. This conversation is resolved. Make sure to revoke the client secret if you haven't already.